src/components/context/context.tsx

```diff
@@ -26,6 +26,7 @@ export interface I18nShape {
     [key: string]: Renderable<object>;
   };
   mappingFunc?: (value: string) => string;
+  render?: (children: any[]) => FunctionComponent<any>;
```

Comment:
I think we should document this fn here is a props docblock at the very minimum (or we could add an entire section to the src-docs view, but I tend to bias towards props docs myself).

```suggestion
  /**
   * Some i18n frameworks don't work with a rendered `<Fragment>` component.
   * The `render` function allows you to pass in another component instead, e.g. `<div>`
   */
  render?: (children: any[]) => FunctionComponent<any>;
```

The above is based on my understanding of this feature/PR, feel free to correct me if I'm wrong!